### DISCUSSION TITLE
Disable Two Thirds toggling?

### DISCUSSION THREAD

**Discussion by eswierk** [rxhanson/Rectangle#462] [Q&A] [answered]
I use the keyboard shortcut for First Two Thirds and Last Two Thirds all the time. So far so good.

What drives me batty is when I absent-mindedly hit First Two Thirds on a window that’s already in First Two Thirds position, and Rectangle decides I must really mean Last Two Thirds. No, I want it to stay in First Two Thirds.

Is there a way to disable this toggling behavior, so that First Two Thirds means First Two Thirds no matter what?

**Comment by rxhanson** [comment, marked as answer]
This behavior can be disabled with the following terminal command followed by an application restart:

defaults write com.knollsoft.Rectangle subsequentExecutionMode -int 2

(https://github.com/rxhanson/Rectangle#halves-to-thirds-repeated-execution-of-half-and-quarter-actions)

This preference is tied to the rest of the Rectangle behavior though, so if you set with that terminal command then you won't get the 1/2 -> 2/3 -> 1/3 cycling behavior or moving across displays on repeated executions.
